README.md

```diff
@@ -223,3 +223,26 @@ use stage 1.
 ```
 ./stage2/bin/zig build --build-file ../build.zig install -Drelease-fast
 ```
+
+## Developing Zig
+
+### Standard Library
+
+First, build the Stage 1 compiler as described in [the Building section](#building).
+Then, make your changes to the standard library files in `std` (note: not
+`build/lib/zig/std`). To test changes to the standard library, do the following
+from the build directory:
+
+1. Run `make install` (on POSIX) or `msbuild -p:Configuration=Release INSTALL.vcxproj`
+   (on Windows).
+2. Test your changes with `bin/zig test lib/zig/std/<changed file>` (e.g. `bin/zig test lib/zig/std/heap.zig`).
```

Comment:
This fails for me with e.g.
```
error: import of file outside package path: '../std.zig'
const std = @import("../std.zig");
            ^
```

What is the recommended solution?

Reply:
Could you provide more about exactly what you're trying? Last I tried, this still works for me.

```sh
cd build
make install
bin/zig test lib/zig/std/heap.zig
```

Reply:
Try with something in a std subfolder e.g. 
```
bin/zig test lib/zig/std/crypto/blake2.zig 
/home/daurnimator/src/zig/build/lib/zig/std/crypto/blake2.zig:6:15: error: unable to find 'test.zig'
const htest = @import("test.zig");
              ^
/home/daurnimator/src/zig/build/lib/zig/std/crypto/blake2.zig:1:13: error: import of file outside package path: '../mem.zig'
const mem = @import("../mem.zig");
            ^
```

Reply:
Ah, ok. I think the alternative would be:

```
bin/zig test ../std/crypto/blake2.zig --override-std-dir ../std
```

but that also fails for me with:
```
/home/ryan/Programming/zig/zig/std/crypto/blake2.zig:1:13: error: import of file outside package path: '../mem.zig'
const mem = @import("../mem.zig");
            ^
```

Got it working by adding `--main-pkg-path ../std`:

```
zig/build$ bin/zig test ../std/crypto/blake2.zig --override-std-dir ../std --main-pkg-path ../std
1/39 test "blake2s224 single"...OK
2/39 test "blake2s224 streaming"...OK
3/39 test "blake2s256 single"...OK
4/39 test "blake2s256 streaming"...OK
5/39 test "blake2s256 aligned final"...OK
6/39 test "blake2b384 single"...OK
7/39 test "blake2b384 streaming"...OK
8/39 test "blake2b512 single"...OK
9/39 test "blake2b512 streaming"...OK
10/39 test "blake2b512 aligned final"...OK
11/39 mem.test "mem.secureZero"...OK
12/39 mem.test "mem.compare"...OK
13/39 mem.test "mem.lessThan"...OK
14/39 mem.test "mem.trim"...OK
15/39 mem.test "mem.indexOf"...OK
16/39 mem.test "comptime read/write int"...OK
17/39 mem.test "readIntBig and readIntLittle"...OK
18/39 mem.test "writeIntBig and writeIntLittle"...OK
19/39 mem.test "mem.tokenize"...OK
20/39 mem.test "mem.tokenize (multibyte)"...OK
21/39 mem.test "mem.separate"...OK
22/39 mem.test "mem.separate (multibyte)"...OK
23/39 mem.test "mem.startsWith"...OK
24/39 mem.test "mem.endsWith"...OK
25/39 mem.test "mem.join"...OK
26/39 mem.test "testStringEquality"...OK
27/39 mem.test "testReadInt"...OK
28/39 mem.test "writeIntSlice"...OK
29/39 mem.test "mem.min"...OK
30/39 mem.test "mem.max"...OK
31/39 mem.test "reverse"...OK
32/39 mem.test "rotate"...OK
33/39 mem.test "asBytes"...OK
34/39 mem.test "toBytes"...OK
35/39 mem.test "bytesAsValue"...OK
36/39 mem.test "bytesToValue"...OK
37/39 mem.test "subArrayPtr"...OK
38/39 mem.test "alignForward"...OK
39/39 mem.test "isAligned"...OK
All tests passed.
```

Reply:
That works for me :)
Maybe we could get this added to the README/wiki?